### DISCUSSION TITLE
Too many arguments to "backpack:add-menu-content"

### DISCUSSION THREAD

**Discussion by charleshrf** [Laravel-Backpack/community-forum#909] [Q&A (Help)] [answered]
I trying to execute this 
` php artisan backpack:add-menu-content "<x-backpack::menu-item title='Pages' icon='la la-file-o' :link=\"backpack_url('page')\" />"`

then this message appears
` Too many arguments to "backpack:add-menu-content" command, expected arguments "code".`

is there steps I miss?
![image](https://github.com/Laravel-Backpack/community-forum/assets/25423128/d690dc9f-a42a-46dd-82f6-60bc1fe65137)

**Comment by karandatwani92** [comment, marked as answer]
Hey @charleshrf 

Please run the command on `CMD` or `Git bash`. Powershell doesn't support this syntax.

**Reply by charleshrf** [reply]
Hi @karandatwani92 

I run the syntax again on `CMD` and it works! Thank you for your help.
